decidim-initiatives/spec/system/print_initiative_spec.rb

```diff
@@ -6,7 +6,7 @@
   context "when initiative print" do
     include_context "when admins initiative"
     let!(:initiative) do
-      create(:initiative, organization: organization, scoped_type: initiative_scope, author: author, state: :accepted)
+      create(:initiative, organization: organization, scoped_type: initiative_scope, author: author, state: :created || :validating)
```

Comment:
```suggestion
      create(:initiative, organization: organization, scoped_type: initiative_scope, author: author, state: :created)
```
Sorry, this doesn't make sense, this is for creating a testing initiative, you need to pass state, this code is just saying if `:create` is `nil`, then put `:validating`. But, :created is a symbol, not nil. Therefore it does nothing.

What you probably want here is to create additional initiatives in different states (but probably this is not the place to do it)

Reply:
Thank you for commenting @microstudi, this was actually a test that was failing after our last commit: https://github.com/decidim/decidim/runs/4925340868?check_suite_focus=true

we thought the test was failing because css ".action-print" was not showing up on the page after we changed the code in show.html.erb file to show progress bar in it's place for published and answered initiatives.